src/app/components/Wallet.tsx

```diff
@@ -658,8 +730,31 @@ class Wallet extends Component<WalletProps, WalletState> {
      * Copy the receive address to the clipboard.
      */
     private copyReceiveAddress(): void {
-        ClipboardHelper.copy(this.state.receiveAddress);
+        const copied = ClipboardHelper.copy(this.state.receiveAddress);
+        if (copied) {
+            this.updateClipboardFeedback("Address copied to clipboard", false);
+        } else {
+            this.updateClipboardFeedback("There was an error copying to clipboard", true);
+        }
+    
     }
+
+    /**
+     * Update Copy Adress to Clipboard feedback 
+     * @param feedback The string to show
+     * @param error An error happened
+     */
+    private updateClipboardFeedback(feedback: string, error: boolean): void {
+        this.setState({
+            clipboardFeedback: feedback,
+            clipboardError: error
+        });
+        const timeOut = 
+        setTimeout(
```

Comment:
Is a good practice to always use `clearTimeout` when the component is going to be destroyed to avoid weird behaviors

Reply:
Additionally, if you click multiple times `copyAddress` the message behaves odd because you are creating N timeouts that will always execute, so it can happen that you display a message that gets cleared inmediately because of an old timeout. You can either check first if there is not a running timeout to trigger it or, my favorite, to clear the timeout if it exists and reinitialize it again

Reply:
@begonaalvarezd There was a fix for those issues in my last push. Can you please check it and let me know if that works?